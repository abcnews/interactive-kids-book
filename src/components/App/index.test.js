const React = require('react');
const renderer = require('react-test-renderer');

const App = require('.');

describe('App', () => {
  test('It renders', () => {
    const scrollyteller = { panels: [] };
    const component = renderer.create(<App scrollyteller={scrollyteller} />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
