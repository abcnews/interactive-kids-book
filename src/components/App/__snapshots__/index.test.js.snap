// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App It renders 1`] = `
<div
  className="root"
>
  <div
    className="AC_W_aNL Block is-richtext is-piecemeal scrollyteller"
  >
    <div
      className="sgMpdhCf zteFRtuv "
    >
      <div />
    </div>
  </div>
</div>
`;
